src/MCPClient/lib/clientScripts/policy_check.py

```diff
@@ -189,7 +210,7 @@ def _file_is_for_access(self):
             return True
         return False
 
-    def _get_manually_normalized_access_derivative_file_uuid(self):
+    def _get_manually_normalized_access_derivative_file_uuid(self) -> File:
```

Comment:
The last line of the method might return `None`, should the return value be `Optional[File]`?

Reply:
oh yes, I miss the exception part. Thanks!